Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -121,17 +134,23 @@ public final class ManagePledgeViewModel:
     )
     .ignoreValues()
 
-    let projectAndReward = project
-      .filterMap { project in
-        guard let backing = project.personalization.backing else {
-          return nil
-        }
+    let userIsCreatorOfProject = project.map {
+      project in project.creator.id == AppEnvironment.current.currentUser?.id
```

Comment:
You could use the shared function `func currentUserIsCreator(of project: Project)` here 😅 

It's in... you guess it, `SharedFunctions.swift`